README.md

```diff
@@ -1,4 +1,5 @@
-# bs-faker [![Build Status](https://travis-ci.org/Schniz/bs-faker.svg?branch=master)](https://travis-ci.org/Schniz/bs-faker) ![BuckleScript binding coverage](https://img.shields.io/badge/binding%20coverage-80%2F148-red.svg)
+# bs-faker [![Build Status](https://travis-ci.org/Schniz/bs-faker.svg?branch=master)](https://travis-ci.org/Schniz/bs-faker) ![BuckleScript binding coverage](https://img.shields.io/badge/binding%20coverage-96%2F148-red.svg)
```

Comment:
I counted and it is `104` bindings, but `getLocale` and `setLocale` are the same (i guess?) so `103`?

Reply:
Yeah it's the same, so `103`

Reply:
Maybe I'm wrong with my countings. I'm thinking about adding `BINDINGS.md` that works like:

```markdown
* `faker.name.firstName`: `BsFaker.Name.firstName`
* `faker.name.lastName`: `BsFaker.Name.lastName`
```

I also thought about checking `faker.js` itself for functions, but I guess that there are functions we don't want to implement, and it can be problematic, but maybe it is okay

then, we can just run a script in CI to match it.